### DISCUSSION TITLE
Checkbox.Group 的 x-reactions 有问题

### DISCUSSION THREAD

**Discussion by chenos** [alibaba/formily#1472] [Q&A 2.x]
见 codesandbox 例子，form.values 正常，但是 checkbox.group 的联动效果有问题

https://codesandbox.io/s/condescending-grothendieck-8drnl?file=/App.tsx

**Comment by janryWang** [comment]
是个Bug，下个版本修复

**Reply by janryWang** [reply]
已发版
